lib/smartrewards_manager.py

```diff
@@ -130,5 +130,6 @@ def smartrewards_check_response(self, response):
         print_msg('Rewards for [{}] is [{}]'.format(addr, eligible_balance))
 
     def add_thousands_spaces(self, a):
-        a = float(a)
+        s = a.replace(',', '.')
```

Comment:
You need to check the user's location first. For users using pt_BR the comma is the default for decimal points, for users using en_US the dot is the default. Must check before doing this replace.

```suggestion
        from locale import localeconv
        dp = localeconv()['decimal_point']
        if dp == ',':
            a = a.replace(',', '.')
        a = float(a)
```

Reply:
Changing and sending now! Thanks for this observation.